client/cmd/dexc/gplimport.go

```diff
@@ -0,0 +1,16 @@
+//go:build gpl
```

Comment:
Was thinking about the actual tag. `eth` would be good to express the purpose, while `lgpl` would convey the consequences.  `ethlgpl` would be a little long.  I'm inclined to say `lgpl` since it's most accurate and important.

Reply:
I thought we were using lgpl and eth was gpl. I must be confused though. Can change to eth anyhow, but I thought this might be a thing for other coins as well. Coins we haven't considered adding yet.

Reply:
Ah, we're using Blue Oak, which is close enough to MIT and ISC and the simpler BSD licenses for government work.

The go-ethereum repository is a mix of gpl and lgpl, with lgpl used for their importable packages, and gpl just for the actual geth cmd app.

So, the build tag should express that by setting the flag, it's going to include lgpl code.

Reply:
Changed to "lgpl".